assets/pages/new-contributions-landing/contributionsLanding.jsx

```diff
@@ -42,10 +43,25 @@ const reactElementId = `new-contributions-landing-page-${countryGroups[countryGr
 
 const selectedCountryGroup = countryGroups[countryGroupId];
 
-const { smallMobileHeader } = store.getState().common.abParticipations;
-const extraClasses = smallMobileHeader
-  ? smallMobileHeader.split('_').filter(x => x !== 'control' && x !== 'notintest')
-  : [];
+const { smallMobileHeaderNotEpicOrBanner } = store.getState().common.abParticipations;
+
+const isFromEpicOrBanner = [
```

Comment:
can we put this in a helper lib somewhere and call it in `abtestDefinitions.js`?

Reply:
(making it a function)

Reply:
Done. You may have a better suggestion for the file name, i've gone for `referrerComponent.js`